mne/realtime/epochs.py

```diff
@@ -314,6 +314,7 @@ def _process_raw_buffer(self, raw_buffer):
         verbose = 'ERROR'
         sfreq = self.info['sfreq']
         n_samp = len(self._raw_times)
+        prev_samp = int(self._find_events_kwargs['min_samples'] - 1)
```

Comment:
okay I couldn't fix it in the limited time I had. Here is my attempt: https://github.com/jasmainak/mne-python/commit/b7f3810fc82e87d88ba01a0b378acd2d1680a229. Indeed, it's getting stuck somewhere but it needs more careful debugging. See if you can make it work without getting Travis angry. Otherwise, I'll take a look over the weekend ...